_vector-search/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Vector search
+nav_order: 1
+has_children: false
+has_toc: false
+nav_exclude: true
+permalink: /vector-search/
+redirect_from:
+  - /vector-search/index/
+  - /search-plugins/vector-search/
+---
+
+# Vector search
+
+Vector search enhances traditional lexical search by encoding text, images, and audio as vectors, enabling similarity search with semantic understanding. OpenSearch unifies traditional search, analytics, and vector search in a single solution. As a vector database, it streamlines AI application development by efficiently storing and retrieving high-dimensional data.
+
+<span class="centering-container">
```

Comment:
Can we move button below key features? It looks a bit odd in the middle of the page/